libs/shared/l10n/src/lib/branding.ftl

```diff
@@ -41,7 +41,11 @@
 -product-mozilla-hubs = Mozilla Hubs
 -product-mdn-plus = MDN Plus
 -product-firefox-cloud = Firefox Cloud
--product-mozilla-monitor = Mozilla Monitor
+-product-mozilla-monitor =
```

Comment:
Note, I want to test this a little more, but this is also contingent on the decision to use text vs an image here ([see Slack thread](https://mozilla.slack.com/archives/C4E0W8B8E/p1722526040889559)).